doc/guides/compile.md

```diff
@@ -117,6 +117,36 @@ set the character via `conf.file_separator`.
 conf.file_separator = '/'
 ```
 
+### Name of library directory
+
+In some environments, the `libmruby.a` file requires a different directory name than `lib`.
+You can be changed to any name by the `conf.libdir_name` accessor.
```

Comment:
My suggestion for the sentence is:
```adoc
This directory can be changed to a different name using the `conf.libdir_name` accessor.
```

Reply:
Thanks for your suggestion. Unfortunately this time the merge was faster than I can check.
I am only converting from Japanese to English using [DeepL](https://www.deepl.com/translator). So I would be very grateful if you could improve the English text.
If you don't mind, could you please create a PR? Thank you very much.

Reply:
Sure, at some point I can go over the whole text, although I am also not a native speaker...

Reply:
Okay, I will not say anything unreasonable.
Please let us know if you have any further concerns.